src/openzaak/components/catalogi/admin/forms.py

```diff
@@ -485,6 +486,14 @@ class CatalogusImportForm(forms.Form):
 
 
 class ZaakTypeImportForm(forms.Form):
+
+    identificatie = forms.CharField(
+        label=_("identificatie"),
+        required=False,
+        help_text=_("Zaaktype name. Leave blank to use imported."),
```

Comment:
```suggestion
        help_text=_("Zaaktype identification. Leave blank to use imported."),
```

It's not a name but rather a unique id number

Reply:
Was supposed to change that but now it is done.